resources/js/processes/modeler/components/inspector/ScreenSelect.vue

```diff
@@ -23,6 +23,14 @@
         </multiselect>
         <small v-if="error" class="text-danger">{{ error }}</small>
         <small v-if="helper" class="form-text text-muted">{{ $t(helper) }}</small>
+        <a
+                v-if="content.id"
+                :href="`/designer/screens/${content.id}/edit`"
```

Comment:
Should this go to `/designer/screen-builder/${content.id}/edit` instead? The current URL seems to open a page to edit only the name, description, and category, and not the screen builder.

Reply:
That is the link I was taken to. Let me check.